tests/org.jboss.tools.batch.ui.bot.test/src/org/jboss/tools/batch/ui/bot/test/AbstractBatchTest.java

```diff
@@ -99,4 +160,30 @@ protected void assertNumberOfProblems(int error, int warning) {
 			throw new AssertionError(message, e);
 		}
 	}
+	
+	private int getUniqueProblemsSize(List<Problem> problems) {
+		List<Problem> uniqueProblems = new ArrayList<Problem>();
+		for (Problem problem : problems) {
+			boolean contains = false;
+			// hardcoded way how to find out that custom class object is in list
+			// TODO implement necessary methods into Reddeer's problem class (equals, hashcode)
```

Comment:
I forgot, thanks. Issue is here https://github.com/jboss-reddeer/reddeer/issues/1468.